cmd/tikv-ctl/src/main.rs

```diff
@@ -2485,6 +2536,183 @@ fn run_ldb_command(cmd: &ArgMatches<'_>, cfg: &TiKvConfig) {
     engine_rocks::raw::run_ldb_tool(&args, &opts);
 }
 
+fn run_sst_dump_command(cmd: &ArgMatches<'_>, cfg: &TiKvConfig) {
+    let mut args: Vec<String> = match cmd.values_of("") {
+        Some(v) => v.map(ToOwned::to_owned).collect(),
+        None => Vec::new(),
+    };
+    args.insert(0, "sst_dump".to_owned());
+    let opts = cfg.rocksdb.build_opt();
+    engine::rocks::run_sst_dump_tool(&args, &opts);
+}
+
+fn print_bad_ssts(db: &str, manifest: Option<&str>, pd_client: RpcClient, cfg: &TiKvConfig) {
+    let mut args = vec![
+        "sst_dump".to_string(),
+        "--output_hex".to_string(),
+        "--command=verify".to_string(),
+    ];
+    args.push(format!("--file={}", db));
+
+    let mut stderr = BufferRedirect::stderr().unwrap();
+    let stdout = BufferRedirect::stdout().unwrap();
+    let opts = cfg.rocksdb.build_opt();
+    match run_and_wait_child_process(|| engine::rocks::run_sst_dump_tool(&args, &opts)).unwrap() {
```

Comment:
It will be much more user-friendly to print out the dumping progress, i.e. getting all SST files up front and doing sst_dump one by one.

Reply:
Maybe as a follow-up task.